spec/rails_helper.rb

```diff
@@ -38,7 +38,7 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
-
+  config.include Capybara::DSL
```

Comment:
interesting that this caused issues for you in the past. glad that we've got it in here for future...